http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20210225_add_zone.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="leonardo" author="willyn" id="add_zone">
+        <addColumn tableName="RUNTIME_CONFIG">
+            <column name="zone" type="varchar(254)">
+                <constraints nullable="true"/>
```

Comment:
I don't think this should be `nullable`. we should populate existing rows with the old default zone, and make this non-nullable